src/REopt.jl

```diff
@@ -100,10 +100,10 @@ const FUEL_DEFAULTS = Dict(
     )
 )
 const EMISSIONS_DECREASE_DEFAULTS = Dict(
```

Comment:
If this is Cambium data, why do we use a const for this instead of getting it from the Cambium API?

Reply:
That's a good point. I think the main argument for keeping it this way is that we can avoid adding an extra API call to every REopt run, since this value should only change whenever Cambium is updated. We can also avoid having to add the API call or an endpoint to the REopt API/web tool.

Reply:
makes sense!